src/apps/categories/entryPoints/categoriesValidation.js

```diff
@@ -10,7 +10,6 @@ export const getCategoryByIdValidation = celebrate({
 export const createCategoryValidation = celebrate({
   body: Joi.object().keys({
     title: Joi.string().required(),
-    managedBy: Joi.string().required(),
```

Comment:
if its optional field that could be sent from the client side u should keep the validation like this :
    managedBy: Joi.string()
    
    "without the required"
    
    but if this field will not be sent from the client side then it makes sense to delete it

Reply:
same as the model above, this should be handled by the backend based on the jwt token of the user